nemo_skills/pipeline/run_cmd.py

```diff
@@ -76,32 +110,87 @@ def run_cmd(
         "--not_exclusive",
         help="If --not_exclusive is used, will NOT use --exclusive flag for slurm",
     ),
+    check_mounted_paths: bool = typer.Option(False, help="Check if mounted paths are available on the remote machine"),
```

Comment:
Btw, id like this flag to be true in general to offer users some safety. But that can be discussed.

Reply:
The problem with having it True is that when you schedule large pipelines, it's often the case that subsequent steps take as input files generated by previous steps. So at scheduling time, they don't exist and thus will cause failure. But they will be available by the time the job runs because of slurm dependencies

Reply:
I see that's a fair point to keep it disabled by default.